src/llama.cpp

```diff
@@ -7162,18 +7162,38 @@ static bool weight_buft_supported(const llama_hparams & hparams, ggml_tensor * w
             } break;
         case GGML_OP_SSM_CONV:
             {
-                // TODO: ggml_ssm_conv(ctx, conv_x, model.layers[il].ssm_conv1d);
-                op_tensor = ggml_ssm_conv(ctx, nullptr, w);
+                // FIXME
+                ggml_tensor * conv_x = ggml_new_tensor_3d(ctx, GGML_TYPE_F32, 12345, w->ne[1], 6789);
+                op_tensor = ggml_ssm_conv(ctx, conv_x, w);
```

Comment:
Should this be the other way around, the convolution/filter as the second argument:
```c++
op_tensor = ggml_ssm_conv(ctx, w, conv_x);
```

Reply:
No, it is important that the weight `w` is in the same position as when used during inference so that the backend `supports_op` can check it. This function is called as `ggml_ssm_conv(ctx, conv_x, model.layers[il].ssm_conv1d)` during inference, so the weight is the second argument.

Reply:
Ah I did not realize that, thanks for clarifying!